staging/src/k8s.io/kubectl/pkg/cmd/create/create_job.go

```diff
@@ -19,6 +19,7 @@ package create
 import (
 	"context"
 	"fmt"
+	"k8s.io/kubectl/pkg/util/completion"
```

Comment:
Could you put this import alphabetically with three ones lower in the list?

Reply:
ok

Reply:
has been modified